tests/clients/test_fenda_ditto_client.py

```diff
@@ -0,0 +1,171 @@
+import pytest
+import torch
+from flwr.common import Config
+
+from fl4health.clients.fenda_ditto_client import FendaDittoClient
+from fl4health.model_bases.fenda_base import FendaModel
+from fl4health.model_bases.sequential_split_models import SequentiallySplitExchangeBaseModel
+from fl4health.parameter_exchange.full_exchanger import FullParameterExchanger
+from tests.clients.fixtures import get_client  # noqa
+from tests.test_utils.models_for_test import FeatureCnn, FendaHeadCnn, HeadCnn
+
```

Comment:
One final test that I would recommend is that we test that assertion of the model structures being the same. You can find an example of testing that an assertion raises in `test_get_parameter_exchanger_with_incorrect_model` if you've never done it before.

Reply:
Added the test + corrected the assertion